app/controllers/api/v1/dependencies_controller.rb

```diff
@@ -25,6 +25,6 @@ def check_gem_count
   end
 
   def gem_names
-    @gem_names ||= params[:gems].split(",")
+    @gem_names ||= (params[:gems] || "").split(",")
```

Comment:
might be worth freezing these strings to save two string allocations? alternatively, instead of splitting an empty string, return a constant empty array?